src/rgw/rgw_op.cc

```diff
@@ -576,23 +576,18 @@ int RGWOp::verify_op_mask()
 
 int RGWOp::do_aws4_auth_completion()
```

Comment:
In the future we might want to replace this with a more generic mechanism like `RGWOp::complete_auth`.